std/format/read.d

```diff
@@ -39,7 +73,11 @@ module std.format.read;
     assert(str.unformatValue!(typeof(null))(spec) == null);
 }
 
-/// Integrals
+/**
+Integrals can be read as signed or unsigned decimal value, as dual,
+octal or hexadezimal number. It's also possible to read integers as raw values.
+Reading as raw values depends on the endianess of the processor.
+ */
```

Comment:
Maybe this comment can be broken down into several smaller comments that are put directly above the code paragraph. E.g:

```d
/// Integrals
unitest
{
       // signed decimal values
       auto str = "123";
       auto spec = singleSpec("%s");
       assert(str.unformatValue!int(spec) == 123);
 
       // hexadecimal values
       str = "ABC";
       spec = singleSpec("%X");
       assert(str.unformatValue!int(spec) == 2748);

     .....
}
```